packages/web/src/scss/components/Checkbox/_Checkbox.scss

```diff
@@ -37,17 +36,26 @@ $_field-name: 'Checkbox';
     background-position: center;
     background-size: contain;
     background-repeat: no-repeat;
+    background-color: form-fields-theme.$inline-field-input-background-color;
+
+    &:hover {
+        background-color: form-fields-theme.$inline-field-input-background-color-hover;
+    }
+
+    &:active {
+        background-color: form-fields-theme.$inline-field-input-background-color-active;
+    }
 
     &:checked {
         color: form-fields-theme.$inline-field-input-color-checked;
         background-image: svg.escape(theme.$input-checked-mark);
-        background-color: form-fields-theme.$inline-field-input-color-checked;
+        background-color: currentcolor;
```

Comment:
Isn't it like this then?

```diff
- color: form-fields-theme.$inline-field-input-color-checked;
  background-image: svg.escape(theme.$input-checked-mark);
+ background-color: form-fields-theme.$inline-field-input-color-checked;
```

Reply:
nope, we need to set the color, because we use it in the border.